### PR TITLE
fix: typo

### PR DESCRIPTION
**Pull request by mr-cal**
Fix typo: `develop ers` -> `developers`

### CODE REVIEW

File: rubric/rubric.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
       "Beginner": "Have not learned how to handle constructive feedback professionally. Pull Request comments are low value.",
       "Advanced Beginner": "Accepts constructive feedback calmly but may need multiple iterations to implement the technical solution. Can catch obvious errors in PRs but may be hesitant to challenge or provide actionable suggestions.",
       "Intermediate": "Accepts constructive feedback well and can implement it quickly in and out of code reviews. Provides quality guidance to Advanced Beginners.",
-      "Advanced": "Accepts constructive feedback with grace and acts on it when appropriate. Provides context-appropriate and actionable feedback to develop ers of all levels. Recognizes their own biases and actively works to correct them. Capable of seeing bugs through reading code in PRs before even pulling down the branch.",
+      "Advanced": "Accepts constructive feedback with grace and acts on it when appropriate. Provides context-appropriate and actionable feedback to developers of all levels. Recognizes their own biases and actively works to correct them. Capable of seeing bugs through reading code in PRs before even pulling down the branch.",
```
Comment (kaiwalyakoparkar) [inline rubric/rubric.json:23]: lgtm

**Review by kaiwalyakoparkar — DISMISSED** [review]

**Comment by kaiwalyakoparkar** [comment]
hey @mr-cal I see changes in 3 files but the expected change should only be `rubric.json` could you please make sure there is change only in `rubric.json`?

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by mr-cal** [comment]
Fixed!

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing!


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Elegant

**Review by suitupstan — APPROVED** [review]
Looks awesome!

**Review by kaiwalyakoparkar — APPROVED** [review]
lgtm

**Pull request merged**